spec/api.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
This file was supposed to be used in my unit tests where I would send a mock HTTP request to the IGDB API where this would be the JSON object that I would request.